### DISCUSSION TITLE
Gigabyte Aorus X570 Mobo Question

### DISCUSSION THREAD

**Discussion by MazzicCron** [Rem0o/FanControl.Releases#634] [Q&A]
I'm not sure if this is an issue I should open here or over at Libre, so I wanted to get y'alls input. I'm running the Aorus X570 motherboard and am seeing some interesting results when trying to work with the fan headers. First, my AIO Pump comes up twice in both the Controls and Speed sections (CPU Fan and Pump respectively) as of the latest update - this is weird but fine. The real issue is three of my headers, System Fan 4, System Fan 5 Pump, and System Fan 6 Pump, seem to be read-only. Consulting my manual they appear to be the exact same four-header inputs and I plugged in the same four-pin fans to them, but whereas CPU Fan, System Fan 1, and System Fan 2 all seem to work just fine, I can't control these at all. 

Two questions: has anyone run into this and would this be more of a Libre issue or a Fan Control issue?

**Comment by AtA3301** [comment]
> 

More of a motherboard/UEFI-BIOS thing, but best to ask over on LibreHardware, as FanControl is just showing what Libre tells it.

**Reply by JRITech** [reply]
I am having a similar issue.  It appears that the ITE IT8792E chip that Gigabyte use to control those fan headers only reports RPM, but is missing fan control for some reason.

![image](https://user-images.githubusercontent.com/104372812/165117824-f6eb79f2-c228-42b1-acd7-8dc7c9f58eee.png)
